snowball.podspec

```diff
@@ -0,0 +1,25 @@
+Pod::Spec.new do |s|
+	s.name = "snowball"
+	s.version = "2014.11.10"
+	s.summary = "Snowball is a small string processing language designed for creating stemming algorithms for use in Information Retrieval."
+	s.homepage = "http://snowball.tartarus.org"
+	s.authors = "Martin Porter", "Richard Boulton"
+	s.license = { :type => 'BSD', :text => <<-LICENSE
+	All the software given out on this Snowball site is covered by the BSD License (see http://www.opensource.org/licenses/bsd-license.html ), with Copyright (c) 2001, Dr Martin Porter, and (for the Java developments) Copyright (c) 2002, Richard Boulton.
```

Comment:
This is the wrong link - Snowball is 3 clause BSD, not 2 clause - for the gory details see:

http://article.gmane.org/gmane.comp.search.snowball/1558

And the new homepage URL is: https://snowballstem.org/